assets/js/modules/analytics/components/common/ExistingGTMPropertyNotice.js

```diff
@@ -27,15 +27,22 @@ import { sprintf, __ } from '@wordpress/i18n';
 import Data from 'googlesitekit-data';
 import { MODULES_ANALYTICS } from '../../datastore/constants';
 import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 const { useSelect } = Data;
 
 export default function ExistingGTMPropertyNotice() {
 	const propertyID = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getPropertyID()
 	);
 
-	const gtmAnalyticsPropertyID = useSelect( ( select ) =>
-		select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID()
+	const isTagManagerAvaliable = useSelect( ( select ) =>
```

Comment:
@hussain-t A typo here:
```suggestion
	const isTagManagerAvailable = useSelect( ( select ) =>
```

Reply:
Updated.